components/post_view/channel_intro_message/messages/default/default.tsx

```diff
@@ -0,0 +1,122 @@
+    },
+    default: {
+        id: t('intro_messages.default'),
+        defaultMessage: '**Welcome to {display_name}!**\\n \\nPost messages here that you want everyone to see. Everyone automatically becomes a permanent member of this channel when they join the team.',
+    },
+    addGroupsToTeam: {
+        id: t('intro_messages.addGroupsToTeam'),
+        defaultMessage: 'Add other groups to this team',
+    },
+    beginning: {
+        id: t('intro_messages.beginning'),
+        defaultMessage: 'Beginning of {name}',
+    },
+
+});
+
+const DefaultIntroMessage = ({
+    channel,
+    usersLimit,
+    enableUserCreation,
+    isReadOnly,
+    teamIsGroupConstrained,
+}: Props) => {
+    let teamInviteLink = null;
+    const renderButtons = !isReadOnly && !isArchivedChannel(channel);
+
+    if (!isReadOnly && enableUserCreation) {
+        teamInviteLink = (
+            <TeamPermissionGate
+                teamId={channel.team_id}
+                permissions={[Permissions.INVITE_USER]}
+            >
+                <TeamPermissionGate
+                    teamId={channel.team_id}
+                    permissions={[Permissions.ADD_USER_TO_TEAM]}
+                >
+                    {teamIsGroupConstrained ? (
+                        <ToggleModalButton
+                            className='intro-links color--link'
+                            modalId={ModalIdentifiers.ADD_GROUPS_TO_TEAM}
+                            dialogType={AddGroupsToTeamModal}
+                            dialogProps={{channel}}
+                        >
+                            {/* MM-46602: convert to compass icon after localization is added */}
+                            <LocalizedIcon
+                                className='fa fa-user-plus'
+                                title={{id: t('generic_icons.add'), defaultMessage: 'Add Icon'}}
+                            />
+                            <FormattedMessage
+                                {...messages.addGroupsToTeam}
+                            />
+                        </ToggleModalButton>
+                    ) : (
+                        <AddMembersButton
+                            showSetHeader={renderButtons}
+                            showBoardsButton={renderButtons}
+                            usersLimit={usersLimit}
+                        />
+                    )}
+                </TeamPermissionGate>
+            </TeamPermissionGate>
+        );
+    }
+
+    return (
+        <>
+            <h2 className='channel-intro__title'>
+                <FormattedMessage
+                    {...messages.beginning}
+                    values={{
+                        name: channel.display_name,
+                    }}
+                />
+            </h2>
+            <p className='channel-intro__content'>
+                <FormattedMarkdownMessage
```

Comment:
could you replace `FormattedMarkdownMessage` with the `FormattedMessage` component and use [rich text formatting](https://formatjs.io/docs/react-intl/components/#rich-text-formatting) instead?